site/pages/create-disruption.tsx

```diff
@@ -14,7 +14,7 @@ interface CreateDisruptionProps {
 
 const getReasonOptions = (): JSX.Element[] => {
     const options: JSX.Element[] = [
-        <option value="" disabled>
+        <option value="" disabled key="">
```

Comment:
Does this still correctly load with 'Choose a reason' as the default?

Reply:
Yeah, a unique key is just needed by react when there's a list of elements, it still displays the same